docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -117,7 +117,7 @@ The [Camunda provided module](https://github.com/camunda/camunda-tf-eks-module)
 
 ```hcl
 module "eks_cluster" {
-  source = "github.com/camunda/camunda-tf-eks-module/modules/eks-cluster"
+  source = "git::https://github.com/camunda/camunda-tf-eks-module/modules/eks-cluster?ref=v2.0.0"
```

Comment:
this doesn't work due to being a subdirectory. See the double slash `//` for the path.

either
```
git::https://github.com/camunda/camunda-tf-eks-module//modules/eks-cluster?ref=v2.0.0
```
or
```
github.com/camunda/camunda-tf-eks-module//modules/eks-cluster?ref=v2.0.0
```

Additionaly, are you 100% sure about `v2.0.0` considering previous versions were without the `v` prefix.